### PR TITLE
GitHub - add release command

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added the ***GitHub-create-release*** command.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 1]
```diff
@@ -2950,6 +2950,53 @@ script:
...
+      required: true
+      secret: false
+    - default: false
+      description: Text describing the contents of the tag.
+      isArray: false
+      name: body
+      required: false
+      secret: false
+    - default: false
+      description: True to create a draft (unpublished) release, false to create a published one.
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:2973]:
```suggestion
      description: Whether to create a draft (unpublished) release.
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 2]
```diff
@@ -2950,6 +2950,53 @@ script:
...
+      - 'True'
+      - 'False'
+      defaultValue: 'True'
+    deprecated: false
+    description: Create a release.
+    execution: false
+    name: GitHub-create-release
+    outputs:
+    - contextPath: GitHub.Release.draft
+      description: Whether the release is draft or not
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:2989]:
```suggestion
      description: Whether the release is a draft.
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 3]
```diff
@@ -2950,6 +2950,53 @@ script:
...
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'True'
+      - 'False'
+      defaultValue: 'True'
+    deprecated: false
+    description: Create a release.
+    execution: false
+    name: GitHub-create-release
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:2986]: once a release is created as a draft, is there an option to publish it?
if not, add it in the command's description as a note

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 4]
```diff
@@ -1611,6 +1611,34 @@ def get_github_get_check_run():
...
+        'draft': argToBoolean(args.get('draft'))
+    }
+    response = http_request('POST', url_suffix=RELEASE_SUFFIX, data=data)
+    release_url = response.get('html_url')
+
+    outputs = {
+        'id': response.get('id'),
+        'draft': response.get('draft'),
+        'html_url': response.get('html_url'),
+        'url': response.get('url')
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1630]: [PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
        'url': response.get('url'),
```

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 5]
```diff
@@ -1611,6 +1611,34 @@ def get_github_get_check_run():
...
 
 
+def create_release_command():
+    args = demisto.args()
+    tag_name = args.get('tag_name')
+    data = {
+        'tag_name': tag_name,
+        'name': args.get('name'),
+        'body': args.get('body'),
+        'draft': argToBoolean(args.get('draft'))
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1621]: [PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
        'draft': argToBoolean(args.get('draft')),
```

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 6]
```diff
@@ -1611,6 +1611,34 @@ def get_github_get_check_run():
...
+    }
+    response = http_request('POST', url_suffix=RELEASE_SUFFIX, data=data)
+    release_url = response.get('html_url')
+
+    outputs = {
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1626]: why aren't we exposing the entire object as output?

**Review by yaakovi — COMMENTED** [review]
add tests (unit-test / playbook)

File: Packs/GitHub/Integrations/GitHub/GitHub.yml (hunk 3 above)
Comment (adi88d) [reply Packs/GitHub/Integrations/GitHub/GitHub.yml:2986]: Its possible to publish the release after it created as draft with [update release request](https://docs.github.com/en/rest/reference/repos#update-a-release) do you think we need to add it as other command?

**Review by yaakovi — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.

**Pull request merged**